style.css

```diff
@@ -2,6 +2,20 @@
 @import url("https://fonts.googleapis.com/css?family=Chakra+Petch&display=swap");
 @import url("https://fonts.googleapis.com/css?family=Source+Sans+Pro&display=swap");
 
+html, body {
+  width: 100%;
+  height: 100%;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+}
+
+* {
```

Comment:
I think we should at least put ```body``` in front of the wildcard. ```style.css``` is basically used in all pages, so we have to be careful about using wildcard like this.

Reply:
I update the wildcard in 1b51660dc4a36322e0d7444b05840d1f2bf6f441